Pipfile

```diff
@@ -7,6 +7,7 @@ name = "pypi"
 djangorestframework = "3.*"
 psycopg = "3.*"
 django-unfold = "*"
```

Comment:
Not sure how I missed this, but we should pin the major versions of django-unfold and django-admin-logs. You can check the major versions by searching in `Pipfile.lock`.

Reply:
I have made this change!